src/actions/Find.ts

```diff
@@ -1,4 +1,5 @@
 import { commands } from "vscode";
```

Comment:
Tbh I'm not entirely sure whether we need it for "scout all", as that just passes the text as a VSCode command arg, but I figure probably doesn't hurt?  And guessing could result in VSCode getting upset if you do it on a gigantic file